README.md

```diff
@@ -90,6 +90,22 @@ or
 pnpm import
 ```
 
+### Create translations
```

Comment:
We should also add the `pnpm build` step :D (Currently, it is also doing the i18n script - we can just mention that it does it, and skip that as a separate step, or wdyt?)